### ISSUE TITLE
Anti-aliasing setting gets reset after game restart

### ISSUE THREAD

**Issue by DarioXN** [Muushy/Sprocket-Feedback#502]
The AA (and possibly other) setting gets reset when restarting the game, needing to be set to something else and then back to the intended one to fix it.

In my case I have it set to TAA but when restarting the game it gets set to something else (don't know what), the difference is noticeable.

**Comment by Muushy** [comment]
Is this still happening?

**Comment by DarioXN** [comment]
Yes, just set it to SMAA, restarted the game, and it was on TAA again, but the menu said SMAA still.
Setting it back to TAA changed nothing other than what the menu said.

**Comment by Muushy** [comment]
Fixed in 0.2.37
